jsonnet/components/control-plane.libsonnet

```diff
@@ -42,6 +42,8 @@ function(params)
               if 'path' in e && e.path == '/metrics/cadvisor' then
                 // Drop cAdvisor metrics with excessive cardinality.
                 {
+                  honorTimestamps: true,
```

Comment:
(suggestion) add a comment explaining the settings. Also should we push the  change upstream eventually?